Library/Homebrew/resource.rb

```diff
@@ -47,6 +47,7 @@ def initialize(name = nil, &block)
     @checksum = nil
     @using = nil
     @patches = []
+    @download_strategy = NoDownloadStrategy
```

Comment:
Is this set elsewhere?

Reply:
This is also set when `Formula::url` is called (which is mandatory in regular formulae), and then the strategy will be deduced from url in [`Resource#url`](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/resource.rb).